lib/tasks/vacuum.rake

```diff
@@ -0,0 +1,20 @@
+namespace :vacuum do
+  desc "Clean up old fiscal year proposals"
+  task old_proposals: :environment do
+    ok_to_act = ENV["OK_TO_ACT"] ? true : false
```

Comment:
Since env vars are always strings, If `ENV['OK_TO_ACT']` were set to "false", I think this would still evaluate to `true` because if any string is present, `if [ANYSTRING]` will return `true` 

Perhaps we should pass an arg to this rake task instead of depending on an env var to set something to true or false?